tests/main/interfaces-checkbox-support/task.yaml

```diff
@@ -0,0 +1,46 @@
+summary: Ensure that the checkbox-support interface works.
+
+details: |
+    The checkbox-support interface grants access to systemd APIs that allow
+    starting and killing processes outside of the sandbox. A Go implementation
+    calling those interfaces is bundled in snapd git repository, in the
+    directory tests/lib/plz-run.
+
+systems:
+    # No way to go-build the test application.
+    - -ubuntu-core-*
+
+prepare: |
+    # This test is susceptible to other tests leaving systemd in an unreloaded
+    # state. Ensure that we start from a clean state by reloading systemd.
+    sudo systemctl daemon-reload
+
+    ( cd "$TESTSLIB"/plz-run && CGO_ENABLED=0 go build )
+    mkdir -p test-snapd-checkbox-support/bin/
+    cp -a "$TESTSLIB"/plz-run/plz-run test-snapd-checkbox-support/bin/plz-run
```

Comment:
you can have a script in the test snap that does:
```
#!/bin/sh

exec $SNAP_COMMON/plz-run "$@"
```
and only copy it to $SNAP_COMMON

Reply:
Hmm, but why?

Reply:
I mean, without prebuilt plz-run we still cannot run this on core. Do you mean we could somehow preserve the built binary?

Reply:
yes, it should be possible to build it in project_prepare, like we do with fakestore/fakedevicesvc and others, it will be preserved in the test environment